tracer/build/_build/Build.Steps.cs

```diff
@@ -630,6 +630,42 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                     }
                 });
 
+    Target CopyNativeFilesForTests => _ => _
+        .Unlisted()
+        .After(Clean)
+        .After(BuildTracerHome)
+        .Executes(() =>
+        {
+            foreach(var projectName in Projects.NativeFilesDependentTests)
+            {
+                var project = Solution.GetProject(projectName);
+                var testDir = project.Directory;
+                var frameworks = project.GetTargetFrameworks();
+                var testBinFolder = testDir / "bin" / BuildConfiguration;
+
+                if (IsWin)
+                {
+                    foreach (var framework in frameworks)
+                    {
+                        var (arch, ext) = GetWinArchitectureAndExtension();
+                        var source = MonitoringHomeDirectory / arch / "datadog_profiling_ffi.dll";
+                        var dest = testBinFolder / framework / "LibDatadog.dll";
+                        CopyFile(source, dest, FileExistsPolicy.Overwrite);
```

Comment:
I think this should just copy for all arches on windows
```suggestion
                        foreach (var arch in WindowsArchitectureFolders)
                        {
                            var source = MonitoringHomeDirectory / arch / "datadog_profiling_ffi.dll";
                            var dest = testBinFolder / framework / "LibDatadog.dll";
                            CopyFile(source, dest, FileExistsPolicy.Overwrite);
                         }
```